### PR TITLE
Attempt to speed-up doctests for F-theory tools

### LINKED ISSUE #2162: Speed-up for doctests of FTheoryTools needed
cf. https://github.com/oscar-system/Oscar.jl/issues/1929

(cc @apturner )

### PR DESCRIPTION
**Pull request by HereAround**
This aims to fix https://github.com/oscar-system/Oscar.jl/issues/2162, as originally raised in https://github.com/oscar-system/Oscar.jl/issues/1929 by @lgoettgens .

cc @apturner

### CODE REVIEW

File: experimental/FTheoryTools/TateModels/attributes.jl [hunk 1]
```diff
@@ -87,7 +87,7 @@ julia> tate_section_a6(t);
 Return the Tate polynomial of the global Tate model.
 
 ```jldoctest
-julia> t = global_tate_model(test_base())
+julia> t = su5_tate_model_over_arbitrary_3d_base()
 Global Tate model over a concrete base
```
Comment (fingolfin) [inline experimental/FTheoryTools/TateModels/attributes.jl]: Doc tests fail, apparently this has a different output:
```suggestion
Global Tate model over a not fully specified base
```

File: test/runtests.jl [hunk 2]
```diff
@@ -87,48 +87,48 @@ include("printing.jl")
 include("PolyhedralGeometry/runtests.jl")
 include("Combinatorics/runtests.jl")
 
-include("GAP/runtests.jl")
-include("Groups/runtests.jl")
+#include("GAP/runtests.jl")
```
Comment (fingolfin) [inline test/runtests.jl]: Hint: you can also do multi line comments in Julia, which give a somewhat smaller diff:
```julia
#= 
This is commented out
This as well.
=#
```

But of course this part of the changes will be removed again later on anyway.

Comment (HereAround) [reply test/runtests.jl]: Thank you. I forgot about this. (But yes, this shall be removed eventually anyways.)

File: experimental/FTheoryTools/TateModels/attributes.jl (hunk 1 above)
Comment (HereAround) [reply experimental/FTheoryTools/TateModels/attributes.jl]: Yes, I am working on fixing the doctests right now. I hope that these changes will already speed-up the doctests, but there are other places which I hope to optimize.

File: experimental/FTheoryTools/WeierstrassModels/attributes.jl [hunk 3]
```diff
@@ -106,8 +116,10 @@ Return the Calabi-Yau hypersurface in the toric ambient space
...
 ```jldoctest
-julia> w = global_weierstrass_model(test_base())
-Global Weierstrass model over a concrete base
+julia> auxiliary_base_ring, (f, g, x) = QQ["f", "g", "x"];
+
+julia> w = global_weierstrass_model(f, g, auxiliary_base_ring, 3)
+Global Weierstrass model over a not fully specified base
 
 julia> cy_hypersurface(w)
 Closed subvariety of a normal toric variety
```
Comment (fingolfin) [inline experimental/FTheoryTools/WeierstrassModels/attributes.jl:128]:
```suggestion
[ Info: Base space was not fully specified. Returning hypersurface in AUXILIARY ambient space.
Closed subvariety of a normal toric variety
```

**Comment by HereAround** [comment]
Judging from https://github.com/oscar-system/Oscar.jl/actions/runs/4565696577/jobs/8057172563?pr=2163, the doctests now pass in a bit more than 20 minutes. That should definitely be an improvement.

File: experimental/FTheoryTools/WeierstrassModels/attributes.jl (hunk 3 above)
Comment (HereAround) [reply experimental/FTheoryTools/WeierstrassModels/attributes.jl:128]: I fixed this (and the other doctests).

**Comment by HereAround** [comment]
Also https://github.com/oscar-system/Oscar.jl/pull/2164#event-8888901023 should help.

**Comment by HereAround** [comment]
Please consider merging this.

**Review by simonbrandhorst — APPROVED** [review]

**Pull request merged**